modules/core/src/lib/constants.ts

```diff
@@ -41,7 +41,7 @@ export const COORDINATE_SYSTEM = {
 
   // Non-geospatial
   CARTESIAN: 0
-};
+} as const;
```

Comment:
long term better to convert these to `enum`.

Reply:
TS enum is incredibly verbose when compiled to JS:

```js

var COORDINATE_SYSTEM;
(function (COORDINATE_SYSTEM) {
    COORDINATE_SYSTEM[COORDINATE_SYSTEM["DEFAULT"] = -1] = "DEFAULT";
    COORDINATE_SYSTEM[COORDINATE_SYSTEM["CARTESIAN"] = 0] = "CARTESIAN";
    COORDINATE_SYSTEM[COORDINATE_SYSTEM["LNGLAT"] = 1] = "LNGLAT";
    COORDINATE_SYSTEM[COORDINATE_SYSTEM["METER_OFFSETS"] = 2] = "METER_OFFSETS";
    COORDINATE_SYSTEM[COORDINATE_SYSTEM["LNGLAT_OFFSETS"] = 3] = "LNGLAT_OFFSETS";
})(COORDINATE_SYSTEM || (COORDINATE_SYSTEM = {}));
```

`as const` is a legitimate solution according to the docs.